frontend/src/app/pages/settings-page/settings-page.component.html

```diff
@@ -27,7 +27,7 @@
                         <input matInput placeholder="E-mail" type="text" [formControl]="userEmail" required readonly>
                         <mat-error *ngIf="userEmail.invalid">{{getUserEmailErrorMessage()}}</mat-error>
                     </mat-form-field>
-
+                    <mat-checkbox [formControl]="skipTutorial">Skip tutorial</mat-checkbox>
```

Comment:
I think it is not needed to have such checkbox in Settings page. Optionally we could make a separate accordeon section for operations like this but I don't think we've got time for this ;)